forem/forem
# Re-enable imageUpload native integration

## What type of PR is this? (check all applicable)


## Description

We had disabled this native integration as we rolled out a relatively big refactor. We were blocked by the Apple review, and given the many moving pieces of the refactor we decided to go ahead with rolling out the web (`forem/forem`) parts of it and re-enable this integration whenever the iOS app update becomes available on the AppStore.

[More details here](https://github.com/forem/forem/pull/15212#discussion_r755507405)

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

1. Using the (latest version) iOS mobile app add `canary1.forem.wtf` instance
2. Sign in and open the Post editor
3. Upload a cover image
4. Upload an image to use in the markdown
5. Both should work as expected using the native image picker

Example: https://canary1.forem.wtf/fdoxyz/mobile-image-picker-fpb

### UI accessibility concerns?

None: UI remains the same but we're using the native image picker instead of the fully web solution

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: The app remains unchanged, except for re-enabling a feature that enhances the image picker on mobile apps.

## [optional] What gif best describes this PR or how it makes you feel?

![WWE undertaker - back to life](https://media.giphy.com/media/b6iVj3IM54Abm/giphy.gif)
